contents/docs/user-guides/data-management.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Data Management
+sidebar: Docs
+showTitle: true
+---
+
+<FeatureAvailability allPlans restrictedPlans={['free', 'openSource']} />
```

Comment:
Let's start documenting here what the restrictions are on certain plans, I'm sure at some point @corywatilo and the website team will include those in this component too.